examples/train_simple_navigation_tf2.py

```diff
@@ -368,20 +356,18 @@ def learn(self, obs, action, reward, done):
         ema_reward = 0
         if options.ema_reward_alpha > 0:
             ema_reward = self.calc_ema_reward()
+        ema_reward = tf.constant(ema_reward)
```

Comment:
if options.ema_reward_alpha == 0, the Tensor's dtype will be tf.int32 and it will cause error when it's added to other float Tensor in tf_learn()